cmd/get/launch_plan.go

```diff
@@ -175,6 +181,17 @@ func getLaunchPlanFunc(ctx context.Context, args []string, cmdCtx cmdCore.Comman
 		return err
 	}
 
+	workflowName := launchplan.DefaultConfig.Workflow
+	var workflowFilteredLps []*admin.LaunchPlan
+	if len(launchplan.DefaultConfig.Workflow) > 0 {
```

Comment:
This is weird. I think when you set the workflow, we should set that as a filter automatically

Reply:
Done